openshift/supervisor.env

```diff
@@ -0,0 +1,2 @@
+SUPERVISOR_PORT=
```

Comment:
Provide safe defaults to mandatory params

Reply:
The defaults are in the template.yaml file.
This file is for parameters to override.

Reply:
I think it didn't work for me..
The deploy.sh failed without actually adding value for the parameter SUPERVISOR_PORT